datadog_checks_dev/datadog_checks/dev/tooling/e2e/docker.py

```diff
@@ -15,9 +16,10 @@
 
 
 class DockerInterface(object):
-    def __init__(self, check, env, config=None, metadata=None, agent_build=None, api_key=None):
+    def __init__(self, check, env, base_package=None, config=None, metadata=None, agent_build=None, api_key=None):
```

Comment:
This and some of the other changes in this file are the changes from #2689 on `docker.py`, right ?

Reply:
Rebased, thanks!